diff --git a/node_modules/@fortawesome/fontawesome-svg-core/index.es.js b/node_modules/@fortawesome/fontawesome-svg-core/index.es.js
index 8d6e43a..f077852 100644
--- a/node_modules/@fortawesome/fontawesome-svg-core/index.es.js
+++ b/node_modules/@fortawesome/fontawesome-svg-core/index.es.js
@@ -1166,6 +1166,7 @@ var attributesParser = function (node) {
       extraAttributes['aria-labelledby'] = config.replacementClass + '-title-' + nextUniqueId();
     } else {
       extraAttributes['aria-hidden'] = 'true';
+      extraAttributes['alt'] = '';
     }
   }
 
@@ -1360,6 +1361,7 @@ function generateLayersText(node, nodeMeta) {
 
   if (config.autoA11y && !title) {
     extra.attributes['aria-hidden'] = 'true';
+    extra.attributes['alt'] = '';
   }
 
   return [node, makeLayersTextAbstract({
@@ -1806,6 +1808,7 @@ var icon = resolveIcons(function (iconDefinition) {
         attributes['aria-labelledby'] = config.replacementClass + '-title-' + nextUniqueId();
       } else {
         attributes['aria-hidden'] = 'true';
+        attributes['alt'] = '';
       }
     }
 
diff --git a/node_modules/@fortawesome/fontawesome-svg-core/index.js b/node_modules/@fortawesome/fontawesome-svg-core/index.js
index 29227df..81a2a86 100644
--- a/node_modules/@fortawesome/fontawesome-svg-core/index.js
+++ b/node_modules/@fortawesome/fontawesome-svg-core/index.js
@@ -1172,6 +1172,7 @@ var attributesParser = function (node) {
       extraAttributes['aria-labelledby'] = config.replacementClass + '-title-' + nextUniqueId();
     } else {
       extraAttributes['aria-hidden'] = 'true';
+      extraAttributes['alt'] = '';
     }
   }
 
@@ -1366,6 +1367,7 @@ function generateLayersText(node, nodeMeta) {
 
   if (config.autoA11y && !title) {
     extra.attributes['aria-hidden'] = 'true';
+    extra.attributes['alt'] = '';
   }
 
   return [node, makeLayersTextAbstract({
@@ -1812,6 +1814,7 @@ var icon = resolveIcons(function (iconDefinition) {
         attributes['aria-labelledby'] = config.replacementClass + '-title-' + nextUniqueId();
       } else {
         attributes['aria-hidden'] = 'true';
+        attributes['alt'] = '';
       }
     }
 
