diff --git a/node_modules/firebase-tools/lib/bin/firebase.js b/node_modules/firebase-tools/lib/bin/firebase.js
index 8130544..6f2fb34 100755
--- a/node_modules/firebase-tools/lib/bin/firebase.js
+++ b/node_modules/firebase-tools/lib/bin/firebase.js
@@ -19,6 +19,16 @@ var handlePreviewToggles = require("../handlePreviewToggles");
 var utils = require("../utils");
 var cmd;
 var logFilename = path.join(process.cwd(), "/firebase-debug.log");
+
+var projectRootDir = cwd || process.cwd();
+    while (!fsutils.fileExistsSync(path.resolve(projectRootDir, "./firebase.json"))) {
+        var parentDir = path.dirname(projectRootDir);
+        if (parentDir === projectRootDir) {
+            return null;
+        }
+        projectRootDir = parentDir;
+    }
+    console log(projectRootDir); process.exit(1);
 logger
     .add(winston.transports.Console, {
     level: process.env.DEBUG ? "debug" : "info",
diff --git a/node_modules/firebase-tools/lib/deploy/hosting/uploader.js b/node_modules/firebase-tools/lib/deploy/hosting/uploader.js
index 34915c8..2bae3f1 100644
--- a/node_modules/firebase-tools/lib/deploy/hosting/uploader.js
+++ b/node_modules/firebase-tools/lib/deploy/hosting/uploader.js
@@ -29,7 +29,7 @@ class Uploader {
             concurrency: options.hashConcurrency || 50,
             handler: this.hashHandler.bind(this),
         });
-        this.populateBatchSize = options.populateBatchSize || 1000;
+        this.populateBatchSize = options.populateBatchSize || 100;
         this.populateBatch = {};
         this.populateQueue = new Queue({
             name: "populateQueue",
diff --git a/node_modules/firebase-tools/lib/detectProjectRoot.js b/node_modules/firebase-tools/lib/detectProjectRoot.js
index fcd2c69..61775fb 100644
--- a/node_modules/firebase-tools/lib/detectProjectRoot.js
+++ b/node_modules/firebase-tools/lib/detectProjectRoot.js
@@ -1,15 +1,16 @@
-"use strict";
-var fsutils = require("./fsutils");
-var path = require("path");
-module.exports = function (cwd) {
-    var projectRootDir = cwd || process.cwd();
-    while (!fsutils.fileExistsSync(path.resolve(projectRootDir, "./firebase.json"))) {
-        var parentDir = path.dirname(projectRootDir);
-        if (parentDir === projectRootDir) {
-            return null;
-        }
-        projectRootDir = parentDir;
+'use strict'
+var fsutils = require('./fsutils')
+var path = require('path')
+module.exports = function(cwd) {
+  var projectRootDir = cwd || process.cwd()
+  while (
+    !fsutils.fileExistsSync(path.resolve(projectRootDir, './firebase.json'))
+  ) {
+    var parentDir = path.dirname(projectRootDir)
+    if (parentDir === projectRootDir) {
+      return null
     }
-    return projectRootDir;
-};
-//# sourceMappingURL=detectProjectRoot.js.map
\ No newline at end of file
+    projectRootDir = parentDir
+  }
+  return projectRootDir
+}
