diff --git a/node_modules/firebase-tools/lib/deploy/hosting/uploader.js b/node_modules/firebase-tools/lib/deploy/hosting/uploader.js
index 34915c8..2bae3f1 100644
--- a/node_modules/firebase-tools/lib/deploy/hosting/uploader.js
+++ b/node_modules/firebase-tools/lib/deploy/hosting/uploader.js
@@ -29,7 +29,7 @@ class Uploader {
             concurrency: options.hashConcurrency || 50,
             handler: this.hashHandler.bind(this),
         });
-        this.populateBatchSize = options.populateBatchSize || 1000;
+        this.populateBatchSize = options.populateBatchSize || 400;
         this.populateBatch = {};
         this.populateQueue = new Queue({
             name: "populateQueue",
