diff --git a/node_modules/react-typography/dist/GoogleFont.js b/node_modules/react-typography/dist/GoogleFont.js
index 59b2a24..0419f76 100644
--- a/node_modules/react-typography/dist/GoogleFont.js
+++ b/node_modules/react-typography/dist/GoogleFont.js
@@ -19,7 +19,6 @@ var GoogleFont = function GoogleFont(props) {
       str += font.name.split(' ').join('+');
       str += ':';
       str += font.styles.join(',');
-
       return str;
     });
 
@@ -27,7 +26,7 @@ var GoogleFont = function GoogleFont(props) {
 
     if (fontsStr) {
       return _react2.default.createElement('link', {
-        href: '//fonts.googleapis.com/css?family=' + fontsStr,
+        href: '//fonts.googleapis.com/css?family=' + fontsStr + '&display=swap',
         rel: 'stylesheet',
         type: 'text/css'
       });
